tests/test-cli/test-config.js

```diff
@@ -77,7 +77,7 @@ describe('config', () => {
   it('call config with no arguments', done => {
     nexpect.spawn('node', [process.cwd() + '/src/cli/bin.js', 'config'])
       .run((err, stdout, exitcode) => {
-        const expected = 'error	argument \'key\' is required'
+        const expected = "error	argument 'key' is required"
```

Comment:
interesting, apparently both vim and node understood that there was a tab char there, but github just ignored it, changing to `\t`:)